testspec.yml

```diff
@@ -8,6 +8,10 @@ phases:
   pre_build:
     commands:
       - start-dockerd
+      - |
+        if expr "${ARCH_TYPE}" : "graviton" >/dev/null; then
+          docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

Comment:
Please add a comment on the buildspec or here in this comment thread about why this statement is necessary.

Reply:
Comment added. This operation might not last long as we might have something else to use in its place.

Reply:
Is this added explicitly for sanity tests?